src/api/app/views/webui/user/_info.html.haml

```diff
@@ -3,30 +3,24 @@
 
```

Comment:
This view now contains the code from *src/api/app/views/webui/user/user_profile_redesign/_info.html.haml*.